gokart/utils.py

```diff
@@ -7,3 +11,48 @@ def add_config(file_path: str):
     _, ext = os.path.splitext(file_path)
     luigi.configuration.core.parser = ext
     assert luigi.configuration.add_config_path(file_path)
+
+
+T = TypeVar('T')
+if sys.version_info >= (3, 10):
+    from typing import TypeAlias
+
+    FlattableItems: TypeAlias = T | Iterable['FlattableItems[T]'] | dict[str, 'FlattableItems[T]']
```

Comment:
[nits] Flattenable(Flatten-able) might be more comprehensive

Reply:
fixed [d58fe04](https://github.com/m3dev/gokart/pull/362/commits/d58fe0495043b9f2d26032e7327872fd29731c4b)